iis-common/src/main/java/eu/dnetlib/iis/common/spark/SparkSessionSupport.java

```diff
@@ -20,49 +20,49 @@ private SparkSessionSupport() {
 
     /**
      * Runs a given job using SparkSession created using default builder and supplied SparkConf. Stops SparkSession
-     * when SparkSession is managed. Allows to reuse SparkSession created externally.
+     * when SparkSession is shared e.g. created in tests. Allows to reuse SparkSession created externally.
      *
-     * @param conf                  SparkConf instance
-     * @param isSparkSessionManaged When true will stop SparkSession
-     * @param job                   Job using constructed SparkSession
+     * @param conf               SparkConf instance
+     * @param sharedSparkSession When true will not stop SparkSession
```

Comment:
Could we follow the convention and rename it to `isSparkSessionShared` or something similar?